.github/workflows/next-build-image.yaml

```diff
@@ -12,12 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: Build Next Image
+name: Build Next and Tag Image
 
 on:
+  schedule:
+    # run at 3:18 UTC every day
+    - cron: "18 3 * * *"
   push:
-    branches:
-      - main
+    tags:
```

Comment:
we previously decided to not push tagged container images in the comunity repo, choosing instead to focus on the downstream Konflux builds. Why do we want to return to tagged images?

Reply:
We need arm images for rhdh-local users and plugin developers using Macs.
We need tagged (versioned) because when developing or testing plugins they need to target a specific version.

Reply:
hmm. OK, seems like going backwards but +1.